cloudsmith/tox.ini

```
@@ -0,0 +1,23 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py39
+envlist =
+    py{39}
```

```suggestion
basepython = py38
envlist =
    py{38}
```
Is what the agent runs atm